tool/site/server_hapi/src/services/communication/live-eps-client.ts

```diff
@@ -52,6 +52,18 @@ export class LiveEpsClient implements EpsClient {
     return {statusCode, fhirResponse, spineResponse: asString(spineResponse)}
   }
 
+  async makeReturnRequest(body: Task): Promise<EpsResponse<OperationOutcome>> {
+    const requestId = uuid.v4()
+    const rawResponseHeaders = {
+      "x-raw-response": "true"
+    }
+    const response = await this.makeApiCall<OperationOutcome>("Task", body, undefined, requestId)
+    const statusCode = response.status
+    const fhirResponse = response.data
+    const spineResponse = (await this.makeApiCall<string | OperationOutcome>("Task", body, undefined, requestId, rawResponseHeaders)).data
```

Comment:
another one for another pr, would be good to only do a spineResponse call if we know that the FHIR response has a 200 (to stop validation error happening twice)